### PR TITLE
[Applab Datasets] Add missing id column, capitalize table names

### PR DESCRIPTION
**Pull request by ajpal**
# Description
1. add id column to spotify and static dataset scripts
2. Capitalize the names of existing static datasets.

image
image
image

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: bin/cron/applab_datasets/spotify [hunk 1]
```diff
@@ -15,7 +15,7 @@ VIRAL_50_WORLD_URL = 'https://spotifycharts.com/viral/global/daily/latest/downlo
 
 def get_top200_data(url)
   records = {}
-  columns = ['Position', 'Track Name', 'Artist', 'Streams', 'URL']
+  columns = ['id', 'Track Name', 'Artist', 'Position', 'Streams', 'URL']
```
Comment (jmkulwik) [inline bin/cron/applab_datasets/spotify:18]: Nit: Not sure whether this matters, but I thought the lower-case `id` looked a little funny next to all the other capitalized column names.

Comment (jmkulwik) [reply bin/cron/applab_datasets/spotify:18]: i.e. could be ID instead.

File: bin/oneoff/applab_datasets/upload_csv_to_firebase [hunk 2]
```diff
@@ -26,7 +26,7 @@ def parse_csv(filename)
     records[id] = record.to_h.to_json
     id += 1
   end
-  return records, table.headers
+  return records, table.headers.unshift('id')
```
Comment (jmkulwik) [inline bin/oneoff/applab_datasets/upload_csv_to_firebase:29]: (not blocking - just for context) What does `.unshift('id')` do here?

**Review by jmkulwik — APPROVED** [review]
LGTM!

File: bin/cron/applab_datasets/spotify (hunk 1 above)
Comment (ajpal) [reply bin/cron/applab_datasets/spotify:18]: I think it's probably better to keep it consistent with every other data table (always lowercase `id`)
If we want to change this across the board for the data tab, that's definitely doable, though.

File: bin/oneoff/applab_datasets/upload_csv_to_firebase (hunk 2 above)
Comment (ajpal) [reply bin/oneoff/applab_datasets/upload_csv_to_firebase:29]: Prepends `'id'` to the front of the list. We want to make sure `'id'` is the first column shown in the table, and this was the easiest way I could think to ensure that.

**Pull request merged**